src/review/llm/feedbackProcessor.ts

```diff
@@ -39,37 +35,29 @@ const createSummary = async (
   return summary;
 };
 
-const rateFeedbacks = async (
-  model: AIModel,
-  feedbacks: IFeedback[]
-): Promise<IFeedbackWithRating[]> => {
-  const ratingPromptWithFeedbacks = ratingPrompt.replace(
-    "{feedback}",
-    JSON.stringify(feedbacks)
-  );
-
-  return await model.callModelJSON<IFeedbackWithRating[]>(
-    ratingPromptWithFeedbacks
-  );
-};
-
 const pickBestFeedbacks = async (
-  model: AIModel,
   feedbacks: IFeedback[],
   limit: number
-): Promise<IFeedbackWithRating[]> => {
-  const ratedFeedbacks = await rateFeedbacks(model, feedbacks);
+): Promise<IFeedback[]> => {
+  // Filter out feedbacks with logafScore of 4 and 5, since we don't need to display them.
+  const filteredFeedbacks = feedbacks.filter(
+    (feedback) => feedback.logafScore < 4
+  );
 
-  const ratingPriorityQueue = new PriorityQueue<IFeedbackWithRating>();
+  // Use the priority queue with some randomization to pick feedbacks to display. This is to avoid showing the same feedbacks every time. We add wights so that feedbacks with lower logafScore are more likely to be picked.
```

Comment:
```suggestion
  // Use the priority queue with some randomization to pick feedbacks to display. This is to avoid showing the same feedbacks every time. We add weights so that feedbacks with lower logafScore are more likely to be picked.
```